pkg/cli/start.go

```diff
@@ -118,9 +119,53 @@ func initInsecure() error {
 	return nil
 }
 
+// maxSizePerProfile is the maximum total size in bytes for profiles per
+// profile type.
+var maxSizePerProfile = envutil.EnvOrDefaultInt64(
```

Comment:
I think we should go for number of profiles, not size.
goroutines especially can get quite big.